src/pages/guides/using/aem/aem_ims_config.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
why adding this here, it's already in the deprecated folder... right ?

Reply:
Yes, looks like this is a miss from my side. Will remove it.